pages/admin/profile/index.js

```diff
@@ -41,12 +44,29 @@ class Profile extends Component {
                         </Button>
                     </Col>
                     <Col span={24}>
-                        {isViewAction ? <ViewProfile /> : <EditProfile />}
+                        { 
+                            isViewAction ? 
+                                <ViewProfile user={user} /> : 
+                                <EditProfile 
+                                    updateUser={updateUser} 
+                                    user={user}    
+                                />
+                        }
                     </Col>
                 </Row>
-            </Layout>
+            </React.Fragment>
         );
     }
 }
 
-export default Profile;
+const mapStateToProps = state => (
+    {
+        user: state.auth.user,
```

Comment:
use a selector to pick the parts of the state that you want

Reply:
True 😅, I guess I have to read about redux-selectors

Reply:
its super easy take a look at (this)[https://github.com/Enye-Academy/Agroeasy/blob/master/src/components/signup/selectors.js]. They are just functions that you pass around and will have consistent behavior in the component and in redux, in terms of accessing piece of the state

Reply:
@teenoh we can sync and I will show you. It's just a function that helps you pick a piece from the state

Reply:
thanks @EngrEric , @Way2nnadi helped me figure it out